src/leds/README.md

```diff
@@ -0,0 +1,84 @@
+# Leds application
+
+The `Leds` application demonstrates how to read and write to the leds device file to control the physical leds on a device.
+
+## Limitations
+
+From the `Host` a path is shared to the `led device file` for a physical port and the path is then mounted in the container when LXC starts the container. 
```

Comment:
Same sed expression as above, and also make sure to reflow _all_ paragraphs in the REDME to not overflow 78 character width limit.